### PR TITLE
added test file for home

### PR DESCRIPTION
**Pull request by Karan555**

### CODE REVIEW

File: src/test/java/com/bjs/learnig/Learning_Maven/HomeTest.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
+package com.bjs.learnig.Learning_Maven;
+
+public class HomeTest {
+
+	public static void paymentapp(){
```
Comment (technaveenks) [inline src/test/java/com/bjs/learnig/Learning_Maven/HomeTest.java:5]: Remove all commented lines

**Review by technaveenks — CHANGES_REQUESTED** [review]
Remove all commented lines

File: src/test/java/com/bjs/learnig/Learning_Maven/HomeTest.java (hunk 1 above)
Comment (Karan555) [reply src/test/java/com/bjs/learnig/Learning_Maven/HomeTest.java:5]: Removed comments

**Comment by technaveenks** [comment]
LGTM

**Pull request merged**